app/reducers/channels.js

```diff
@@ -44,19 +43,113 @@ export const RECEIVE_SUGGESTED_NODES_ERROR = 'RECEIVE_SUGGESTED_NODES_ERROR'
 export const RECEIVE_SUGGESTED_NODES = 'RECEIVE_SUGGESTED_NODES'
 
 // ------------------------------------
-// Actions
+// Helpers
 // ------------------------------------
-export function setChannelForm(form) {
-  return {
-    type: SET_CHANNEL_FORM,
-    form
+
+/**
+ * Get a name to display for the channe.
+ *
+ * This will either be:
+ *  - the alias of the node at the othe end of the channel
+ *  - a shortened public key.
+ * @param  {Object} channel Channel object.
+ * @param  {Array} nodes Array of nodes.
+ * @return {SAtring} Channel display name.
+ */
+const getDisplayName = (channel, nodes) => {
+  const remoteNodePubkey = getRemoteNodePubKey(channel)
+  const node = nodes.find(n => n.pub_key === remoteNodePubkey)
+
+  if (node && node.alias && node.alias.length) {
+    return node.alias
+  }
+
+  return remoteNodePubkey.substring(0, 10)
```

Comment:
An util for this is in `next` I suppose ?

Reply:
Well, `getDisplayName` is the utility function which in the next branch I have used to replace the multiple instances where we are calling `.substring(0, 10)` on the channel pubkey.

So this will be the one remaining instance of this call.